cli/src/semgrep/app/scans.py

```diff
@@ -382,7 +380,7 @@ def report_findings(
                 click.echo(f"Server returned following warning: {message}", err=True)
 
             if "task_id" in res:
-                complete["task_id"] = res["task_id"]
+                complete.task_id = res["task_id"]
```

Comment:
You called `complete.to_json` earlier, so this probably won't be effective anymore right?

Reply:
yep, fixed